From 0062f13b1cf3af04ee8d13e737f878666fb95263 Mon Sep 17 00:00:00 2001
From: pafonsoX <philippex.afonso@intel.com>
Date: Fri, 14 Feb 2014 14:48:46 +0100
Subject: [PATCH 7/8] [REVERT ME][AUDIO] create header file for Intel features

BZ: 217971

Orig-Bug: 166558

Features like LPAL and Context Awareness are specific to Intel.
So a new header file will be helpful for future Android code
rebase.

This patch includes a partial revert of some old patches :
90345, 116628, 149803.
It is enough to have the defines in audio-intel.h.

Change-Id: Idfea315682193dc3f03c8116608b7393fb7b158b
Category: feature differentiation
Domain: AudioComms-VoiceTrigger
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: pafonsoX <philippex.afonso@intel.com>
---
 include/hardware/audio.h       |  2 ++
 include/hardware/audio_intel.h | 37 +++++++++++++++++++++++++++++++++++++
 2 files changed, 39 insertions(+)
 create mode 100644 include/hardware/audio_intel.h

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 2700c6e..8256d5e 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -29,6 +29,8 @@
 #include <system/audio.h>
 #include <hardware/audio_effect.h>
 
+#include "audio_intel.h"
+
 __BEGIN_DECLS
 
 /**
diff --git a/include/hardware/audio_intel.h b/include/hardware/audio_intel.h
new file mode 100644
index 0000000..00d44b7
--- /dev/null
+++ b/include/hardware/audio_intel.h
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2014 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ANDROID_AUDIO_HAL_INTEL_INTERFACE_H
+#define ANDROID_AUDIO_HAL_INTEL_INTERFACE_H
+
+/* Always Listening/VTSV */
+#define AUDIO_PARAMETER_KEY_ALWAYS_LISTENING_STATUS "vtsv_active"
+#define AUDIO_PARAMETER_VALUE_ALWAYS_LISTENING_ON "true"
+#define AUDIO_PARAMETER_VALUE_ALWAYS_LISTENING_OFF "false"
+
+/* Always Listening Route/VTSV */
+#define AUDIO_PARAMETER_KEY_ALWAYS_LISTENING_ROUTE "vtsv_route"
+#define AUDIO_PARAMETER_KEY_LPAL_DEVICE "lpal_device"
+#define AUDIO_PARAMETER_VALUE_ALWAYS_LISTENING_ROUTE_ON "on"
+#define AUDIO_PARAMETER_VALUE_ALWAYS_LISTENING_ROUTE_OFF "off"
+
+/* Context awareness */
+#define AUDIO_PARAMETER_KEY_CONTEXT_AWARENESS_STATUS "context_awareness_status"
+#define AUDIO_PARAMETER_VALUE_CONTEXT_AWARENESS_ON "on"
+#define AUDIO_PARAMETER_VALUE_CONTEXT_AWARENESS_OFF "off"
+
+
+#endif  // ANDROID_AUDIO_INTERFACE_H
\ No newline at end of file
-- 
1.9.2

